src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -121,27 +122,25 @@ protected AbstractFindUsagesService(IThreadingContext threadingContext)
             cancellationToken.ThrowIfCancellationRequested();
 
             // Find the symbol we want to search and the solution we want to search in.
-            var symbolAndSolutionOpt = await FindUsagesHelpers.GetRelevantSymbolAndSolutionAtPositionAsync(
+            var symbolAndProjectOpt = await FindUsagesHelpers.GetRelevantSymbolAndProjectAtPositionAsync(
```

Comment:
note: it's unortunate that hte host side still *starts* with a symbol.  However, this is something that is harder to change now.  The reason for this is metadata-as-source.  The way MAS works is that you start with teh MAS document in the MAS workspace.  We find the symbol you're on, then map that back to the normal metadata symbol in the primary workspace.  We then do a FAR on that metadata symbol.

@tmat We could likely support this in OOP as well.  But it means OOP understanding more about different workspaces (and especially providing the services so those workspaces can map between each other).  This was out of scope for this PR, but would be good to have in the future.